latex/ledger-spec.tex

```diff
@@ -1009,21 +1174,29 @@ \subsection{Witnesses}
 The witnessed transition rule, in fact, defines the same UTxO update as the
 non-witnessed rule. Essentially, the witnessed rule says we may only apply the
 the non-witnessed rule whenever the following
-additional precondition is satisfied
+additional preconditions are satisfied
 (stated in Rule~\ref{eq:utxo-witness-inductive} in
 Figure~\ref{fig:rules:utxow}):
 
 \begin{itemize}
  \item For each input $\var{i}$ of the signal transaction $\var{tx}$,
  there exists a verifiable witness (there could be more than one valid
- key-signature pair serving as a witness) such the hash of the spending key
- in the address corresponding to the output of $\var{i}$ in the current UTxO
+ key-signature pair serving as a witness) such that the hash of the spending key
+ of the address corresponding to the output of $\var{i}$ in the current UTxO
  is the same as the hash of the verification key of the witness.
+
+ \item For every withdrawal request address of $\var{tx}$,
+ there exists a verifiable witness such the hash of the spending key of the
+ withdrawal address is the same as the hash of the verification key of the witness.
 \end{itemize}
 
+\begin{todo}
+  Why do the reward requests need witnessing?
```

Comment:
Nothing very bad would happen if an unauthorized withdrawal occurred, but the owner should still be in control.